openllm-python/src/openllm/_service.py

```diff
@@ -69,6 +71,90 @@ async def generate_stream_v1(input_dict: dict[str, t.Any]) -> t.AsyncGenerator[s
   else:
     return runner.generate_iterator.async_stream(qa_inputs.prompt, adapter_name=qa_inputs.adapter_name, echo=echo, **qa_inputs.llm_config.model_dump())
 
+@svc.api(route='v1/completions', input=_JsonInput, output=bentoml.io.Text())
+async def completion_v1(input_dict: dict[str, t.Any], ctx: bentoml.Context):
+  stream = input_dict.pop('stream', False)        # Determine whether to stream the response
```

Comment:
are you using tab here? I think it should just be two spaces.

Reply:
Yes, sure I'll do that.